sdk/tests/core/test_az_mqtt5_rpc_client_codec.c

```diff
@@ -45,16 +45,6 @@
   "vehicles/" TEST_MODEL_ID "/commands/" TEST_SERVER_ID "/" TEST_COMMAND_NAME "/" TEST_CLIENT_ID \
   "\0"
 
-static void test_az_mqtt5_rpc_client_codec_options_default_success(void** state)
-{
-  (void)state;
-
-  az_mqtt5_rpc_client_codec_options options = az_mqtt5_rpc_client_codec_options_default();
-
-  assert_int_equal(options.subscribe_timeout_in_seconds, AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS);
```

Comment:
Should this test be kept for the other default values? (subscription_topic_format and request_topic_format)

Reply:
The following test (`...init_no_options_succeed`) already tests the default values